### ISSUE TITLE
[css-paint-api] Ed Issue: Make note that the contents of custom paint are not accessible.

### ISSUE THREAD

**Issue by bfgeek** [w3c/css-houdini-drafts#21]
Custom paint contents are not accessible to things like screen-readers (for example drawText). Authors should they communicate any useful information through regular a11y APIs.

**Comment by bfgeek** [comment]
Fixed in 6b11b7d22e5dae90bfae49fa7c58f420c9d08769.
